README.md

```diff
@@ -114,6 +117,52 @@ module.exports = {
 }
 ```
 
+#### Hot Module Reloading (HMR)
+
+extract-mini-css-plugin supports hot reloading of actual css files in development. Some options are provided to enable HMR of both standard stylesheets and locally scoped css or CSS modules. Below is an example configuration of mini-css for HMR use with CSS modules
+
+**webpack.config.js**
+
+```js
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+module.exports = {
+  plugins: [
+    new MiniCssExtractPlugin({
+      // Options similar to the same options in webpackOptions.output
+      // both options are optional
+      filename: "[name].css",
+      chunkFilename: "[id].css"
+    })
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.css$/,
+        use: [
+          {
+            loader: MiniCssExtractPlugin.loader,
+            options: {
+              // only enable hot in development
+              hot: process.env.NODE_ENV === 'development',
+              // modules enables hmr of css modules which
+              // need to be handled differently compared to typical css files
+              modules: true,
```

Comment:
Can we check `locals` and enable this by default if `css-loader` return locals https://github.com/webpack-contrib/css-loader/blob/master/src/index.js#L326?

Reply:
Ive had quite a few issues with this, but ill release something and see if anyone complains